mycore-sword/src/main/java/org/mycore/sword/MCRSwordUtil.java

```diff
@@ -572,10 +576,6 @@ static void addEditMediaLinks(String collection, DepositReceipt depositReceipt,
         }
     }
 
-    public interface MCRFileValidator {
```

Comment:
Why has this public interface  been removed while fixing `AvoidReassigningParameters`?

Reply:
Just checked it. It seems to be in line 376 now. I am not sure why it is marked as deleted here but it is still there.